src/Juvix/Core/IR/Evaluator.hs

```diff
@@ -9,48 +11,86 @@ import Juvix.Core.IR.Types
 import Juvix.Core.Types
 import Juvix.Library hiding (show)
 
+-- TODO: extend functions to annotated terms:
```

Comment:
What is required to do this? It looks like it's already under a `∀ ext`? I guess `Value` discards the annotations though.

Reply:
oops, sorry, that's a comment reminding me to add the `∀ ext` that I forgot to remove.